pkg/plugin/manager.go

```diff
@@ -88,6 +89,29 @@ func (m *Manager) Install(resource string, installOpts *InstallOpts) (err error)
 	return nil
 }
 
+// InstallDefaultPlugin installs the default core plugin bundled with the wrapper cli if something
+// went wrong with installing the plugin from Cosmos
```

Comment:
For consistency this should rather stay an implementation detail under the `setup` package, the plugin package has no dependency on cosmos.